lua/entities/seamless_portal.lua

```diff
@@ -348,6 +382,8 @@ if CLIENT then
 			-- invert mouse x
 			hook.Add("InputMouseApply", "portal_flip_scene", function(cmd, x, y, ang)
 				if LocalPlayer():WaterLevel() < 3 then
+					-- What the fuck is this?
+					-- TODO: Comment what this does
```

Comment:
I have to invert the mouse movement if the player is underwater, dont ask me why its probably a water shader thing